server/requirements.txt

```diff
@@ -8,7 +8,7 @@ flask-compress==1.13
 flask-cors==5.0.0
 flask-restful==0.3.9
 flask-talisman==1.0.0
-flatbuffers>=1.11.0,<2.0.0 # cellxgene is not compatible with 2.0.0. Requires migration
+flatbuffers==2.0.7
```

Comment:
according to comment, flatbuffer 2.0 requires migration. note that you might not get an error after upgrading to 2.0.7 because flatbuffers is only used as part of the dev cycle to generate the autogenerated flatbuffer code. i would not touch this requirement if possible.

Reply:
So that means no Python 3.12 - if so we can scrap this PR and stick to 3.11

Reply:
btw - version 2.0.0 doesn't work with Python 3.11 and throws 

```
TypeError: Builder.EndVector() takes 1 positional argument but 2 were given
```

and in 3.12 it throws an error because it is looking for `imp` module which is dropped in this version of python
on some of the unit tests

2.0.7 already updated to use `importlib` and no changes needed outside the ones in this PR 

Could it be that this comment was made against flatbuffers==2.0.0 which is significantly different than 2.0.7?

Unfortunately, flatbuffers only started logging their changes in [2.0.7](https://github.com/google/flatbuffers/blob/master/CHANGELOG.md)

Reply:
Can we add back a comment to 2.0.7 letting future devs know that the flatbuffer generation has not been tested on any version > 2.0.0 so some debugging might be required?

Reply:
Yes, great idea! Thanks @atarashansky 🙏🏽